### ISSUE TITLE
Will router.js be maintained in the near future?

### ISSUE THREAD

**Issue by adrian-moisa** [tildeio/router.js#234]
Sounds like a dummy question but let's review some context. I have been working with angularjs, angular2 and react in the previous years and all of them have popular routing solutions. Now I'm trying to ditch frameworks in favor of native web components. So far I managed to build a basic demo. Now I need to setup a proper routing solution. I've been looking around for possible solutions and I have found [Visionmedia page](https://github.com/visionmedia/page.js), [Flatiron director](https://github.com/flatiron/director) and [App-router](https://github.com/erikringsmuth/app-router). All of them seem to have been abandoned by the maintainers. for more than 2 years. I am looking for a router that will be around for the foreseeable future. 

So basically I need some reassurance about the future of tildeio router.js. The project I build I expect it to be alive for several years. Looking at the contribution graphs it looks like the repository is really healthy.

**Comment by rwjblue** [comment]
This repo contains the underlying router library used by Ember for quite some time now. We definitely are not planning on abandoning.

**Comment by adrian-moisa** [comment]
Thank you for the quick answer. I'm glad to hear this!
